src/libs/ajax/Groups.ts

```diff
@@ -84,6 +92,11 @@ export const Groups = (signal?: AbortSignal) => ({
         const res = await fetchSam(`${resourceRoot}/action/use`, _.merge(authOpts(), { signal }));
         return res.json();
       },
+
+      getSupportSummary: async (): Promise<GroupSupportSummary> => {
+        const res = await fetchSam(`api/admin/v1/groups/${groupName}/supportSummary`, _.merge(authOpts(), { signal }));
+        return res.json();
```

Comment:
For viewing JSON, we do have a 3rd party library that we use. 

import ReactJson from '@microlink/react-json-view';

Reply:
Example usage: https://github.com/DataBiosphere/terra-ui/blob/e41f8a1d83e4d9b01f14fda611b862c678577458/src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js#L280